components/backdrop/backdrop.js

```diff
@@ -177,7 +177,7 @@ function hideAccessible(target) {
 
 	let parent = getComposedParent(target);
 	while (parent !== document.documentElement) {
-		if (parent.nodeType === Node.ELEMENT_NODE) {
+		if (parent.nodeType === Node.ELEMENT_NODE && parent !== null) {
```

Comment:
```suggestion
		if (parent !== null && parent.nodeType === Node.ELEMENT_NODE) {
```
Should this be switched?